### DISCUSSION TITLE
Support mix type on a filed

### DISCUSSION THREAD

**Discussion by hengfeiyang** [zinclabs/zinc#145] [General]
Now, one field once set as a type, can't index other type for the field, like this:

```json
{
    "name": "abc",
    "age": 18
}
```

```json
{
    "name": "abd",
    "age": "21"
}
```

This will panic.

Should we support mix type on a filed?

**Comment by GeekRu1** [comment]
Like elasticsearch that way,text type with field of keyword will be useful.Once i want its analysis tokens for searching, but also want to do some aggregations with it.
